src/ImageSharp/Formats/Webp/Lossy/Vp8EncIterator.cs

```diff
@@ -754,7 +755,9 @@ public void MakeChroma8Preds()
 
         public void SwapOut()
         {
+#pragma warning disable IDE0180 // Use tuple to swap values
```

Comment:
Maybe we could add an extra overload to `RuntimeUtility.Swap`?

Reply:
You mean one that takes a `T[]`? The problem I found was that

```c#
RuntimeUtility.Swap(this.YuvOut, this.YuvOut2);
```

Would not work and creating local field references requires a lot more code than we have already.